packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -30,7 +30,7 @@ export const SamlAttributesSchema = z.object({
 
 export const baseRequestSchema = z.object({
   id: z.string(),
-  cxId: z.string(),
+  cxId: z.string().optional(),
```

Comment:
making optional because requests from the outside world will not have a cxID

Reply:
Let's then use a diff base schema for those? It will be noisy/cumbersome to have optional cxId for our internal code when we know it always must be present.